app/models/invoice.rb

```diff
@@ -8,4 +8,12 @@ class Invoice < ApplicationRecord
   has_many :items, through: :invoice_items
 
   validates :status, presence: true
+
+  def items_belonging_to
+    invoice_items.joins(:item).select('invoice_items.status, invoice_items.quantity, invoice_items.id as invoice_item_id, items.*')
+  end
+
+  def total_revenue
+    invoice_items.sum('unit_price * quantity').to_f / 100
+  end
```

Comment:
For consistency, we should consider doing the conversion to dollars in the view.

Reply:
yea scott showed me the method number_to_currency which is cool